### ISSUE TITLE
Some problem about UCF101-24 annotations

### ISSUE THREAD

**Issue by CKK-coder** [joslefaure/HIT#8]
Thank you for your work!
During the inference or val, the actor bounding box is from the detection result of YOWO? If so, could you provide the corresponding val annotations? 
Thank you very much！

**Comment by joslefaure** [comment]
You mean if I can provide the bounding box results of YOWO?

**Comment by ping-High** [comment]
yes

**Comment by joslefaure** [comment]
Sure. Here is the link to one of our preliminary results on UCF-24. [UCF results](https://drive.google.com/file/d/1oRJ_KijZd-McTqgHh0F-O9Ubj2dc9PvW/view?usp=sharing).
The format of the file is: `video_name, timestamp, x1, y1, x2, y2, class_id, confidence_score`

**Comment by ping-High** [comment]
OK! Thank you very much

**Comment by yan-ctrl** [comment]
> OK! Thank you very much

Hello, do you have this dataset? Can you share it with me?

**Comment by yan-ctrl** [comment]
Hello, I want to ask you some questions about the recurrence of the JHMDB dataset:

1) Have you divided the test set and verification set into the same. At the end of the training, the test set is reasoned, and then the map value is calculated according to the real value

2) I find that the loss has been 0 when the iteration reaches 2000 times. Does it mean that the network itself is good enough and the accuracy of the training set is 100%. But why is the test set only about 80%.

3) The loss can be reduced to 0. Does it mean that the improvement of the network has no effect?Does it mean that improving the network has no effect? It seems to be true. I tried to add different attention modules after the 3 * 3 convolution of the basic residual block, and the effect did not improve, but decreased.Can you give me some suggestions? Whether training should be conducted in other data sets.

**Comment by joslefaure** [comment]
1. There is only train and test set. Where you see val set, it actually
means test set. The dataset is divided into train/val according to the
standard approach used by other researchers.

2. The network converges quite fast, and it certainly overfits. I did not
spend any effort tuning the hyperparameters, but you can try it. The reason
why the accuracy of the test set will never be close to that of the train
set is that we test with bounding boxes detected by off-the-shelf
detectors. The mAP on the test set depends a great deal on the correctness
of these bounding boxes, whereas the accuracy of the training set only
depends on the class.

3. As I said in (2), there is overfitting (jhmdb is very small), therefore
I doubt adding to the network will help with the accuracy. Smaller loss
does not necessarily mean better generalization.
Suggestions : try tuning the learning-rate related parameters, the number
of epochs (or test on different checkpoints), other params such as
NUM_PERSON, NUM_OBJECT, NUM_KEYPOINTS can also be changed

**Comment by yan-ctrl** [comment]
![$2(IU@C~3PD0JITK26OJ{3B](https://user-images.githubusercontent.com/74650283/219263127-863eaffe-7b2a-4c76-9239-6f4a07ae4390.png)
Thanks for your patience in replying, I have suspected the overfitting of the network before as it converges too quickly, but dropping to 0 is a bit strange. So could you have a look at the image I uploaded? There are two losses in the printout log，loss_pose_action and total_loss，There are other values inside the brackets, this value is constantly decreasing, what does this value mean, is it the true value of the loss?

**Comment by ping-High** [comment]
> > OK! Thank you very much
> 
> Hello, do you have this dataset? Can you share it with me?
I prepare UCF101-24 datasets as YOWO, if you need them, I can send you by email.

**Comment by yan-ctrl** [comment]
> > > OK! Thank you very much
> > 
> > 
> > Hello, do you have this dataset? Can you share it with me?
> > I prepare UCF101-24 datasets as YOWO, if you need them, I can send you by email.
Yes, I need UCF101-24 data set. Thank you very much. This is my email: 1263989216@qq.com Or you can give me a link to Baidu Cloud, Google Cloud or Alibaba Cloud disk. Alibaba Cloud disk is free and very fast.

**Comment by ping-High** [comment]
> > > > OK! Thank you very much
> > > 
> > > 
> > > Hello, do you have this dataset? Can you share it with me?
> > > I prepare UCF101-24 datasets as YOWO, if you need them, I can send you by email.
> > > Yes, I need UCF101-24 data set. Thank you very much. This is my email: [1263989216@qq.com](mailto:1263989216@qq.com) Or you can give me a link to Baidu Cloud, Google Cloud or Alibaba Cloud disk. Alibaba Cloud disk is free and very fast.

This UCF24 Datasets from YOWO, BaiduYunDisk Link: https://pan.baidu.com/s/11GZvbV0oAzBhNDVKXsVGKg  Password: hmu6

**Comment by yan-ctrl** [comment]
> > > > > OK! Thank you very much
> > > > 
> > > > 
> > > > Hello, do you have this dataset? Can you share it with me?
> > > > I prepare UCF101-24 datasets as YOWO, if you need them, I can send you by email.
> > > > Yes, I need UCF101-24 data set. Thank you very much. This is my email: [1263989216@qq.com](mailto:1263989216@qq.com) Or you can give me a link to Baidu Cloud, Google Cloud or Alibaba Cloud disk. Alibaba Cloud disk is free and very fast.
> 
> This UCF24 Datasets from YOWO, BaiduYunDisk Link: https://pan.baidu.com/s/11GZvbV0oAzBhNDVKXsVGKg Password: hmu6

Ok, thank you for sharing. Do you have the processed UCF101-24 dataset based on HIT? If you have, can you share it with me again? Thank you very much.

**Comment by ping-High** [comment]
> > > > > > OK! Thank you very much
> > > > > 
> > > > > 
> > > > > Hello, do you have this dataset? Can you share it with me?
> > > > > I prepare UCF101-24 datasets as YOWO, if you need them, I can send you by email.
> > > > > Yes, I need UCF101-24 data set. Thank you very much. This is my email: [1263989216@qq.com](mailto:1263989216@qq.com) Or you can give me a link to Baidu Cloud, Google Cloud or Alibaba Cloud disk. Alibaba Cloud disk is free and very fast.
> > 
> > 
> > This UCF24 Datasets from YOWO, BaiduYunDisk Link: https://pan.baidu.com/s/11GZvbV0oAzBhNDVKXsVGKg Password: hmu6
> 
> Ok, thank you for sharing. Do you have the processed UCF101-24 dataset based on HIT? If you have, can you share it with me again? Thank you very much.

sry, I just use UCF on my experiment, I haven't reproduced HIT at present.

**Comment by yan-ctrl** [comment]
OK, that's all right. Thank you.
